_opensearch/install/docker.md

```diff
@@ -384,3 +384,112 @@ The environment variable `"DISABLE_SECURITY_DASHBOARDS_PLUGIN=true"` disables th
 You can't reverse this step as the security dashboards plugin is removed in the process.
 To re-enable security for OpenSearch Dashboards, start a new container and set `DISABLE_SECURITY_DASHBOARDS_PLUGIN` to false or leave it unset.
 {: .note}
+
+## Using Windows PowerShell to Install and Configure Window Subsystem for Linux (WSL)
```

Comment:
Please use sentence case here: Using Windows PowerShell to install and configure Windows Subsystem for Linux (WSL)